### PR TITLE
changed some script to a more robust and universal logic

### PR DESCRIPTION
**Pull request by nicosammito**

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -3,9 +3,11 @@
   "version": "0.0.0",
   "description": "A simple template for a custom React component library",
   "scripts": {
+    "dev": "npm run dev-storybook",
+    "build": "npm run clear && npm run rollup",
+    "clear": "concurrently --kill-others \"rimraf ./dist\" \"rimraf ./storybook-static\"",
     "rollup": "rollup -c",
-    "test": "jest",
-    "storybook": "storybook dev -p 6006",
+    "dev-storybook": "storybook dev -p 6006",
```
Comment (Taucher2003) [inline package.json:10]:
```suggestion
    "storybook:dev": "storybook dev -p 6006",
```

File: package.json [hunk 2]
```diff
@@ -3,9 +3,11 @@
...
     "build-storybook": "storybook build"
```
Comment (Taucher2003) [inline package.json:11]:
```suggestion
    "storybook:build": "storybook build"
```

File: package.json [hunk 3]
```diff
@@ -3,9 +3,11 @@
   "version": "0.0.0",
   "description": "A simple template for a custom React component library",
   "scripts": {
+    "dev": "npm run dev-storybook",
```
Comment (Taucher2003) [inline package.json:6]:
```suggestion
    "dev": "npm run storybook:dev",
```

File: package.json (hunk 1 above)
Comment (nicosammito) [reply package.json:10]: seems like a goof alternative solution 👍🏻

**Review by Taucher2003 — APPROVED** [review]

**Pull request merged**